docs/en/latest/discovery.md

```diff
@@ -92,6 +95,11 @@ Then implement the `_M.init_worker()` function for initialization and the `_M.no
   end
 
 
+  function _M.dump_data()
+      ... ...
```

Comment:
I think an example return data shoul be written here.

Reply:
for example ?
```lua
  function _M.dump_data()
      return {config = your_config, services = your_services, other = ... } 
  end
```